recipes-azure-iot/iot-hub-device-update/iot-hub-device-update/swupdate_v1_grub.sh

```diff
@@ -94,13 +88,16 @@ fi
 
 if [[ $action == "apply" ]]; then
     echo "Applying update." >> "${log_dir}/swupdate.log"
-    mount_boot && grub-editenv /boot/EFI/BOOT/grubenv set omnect_validate_update_part=${update_part} && umount /boot
+    if [ ! -f "/run/omnect-bootloader-update" ]; then
```

Comment:
like the new temporary file name. would you like to change the u-boot file name (/run/bootloader/update) as well?

Reply:
done